HPCC-15301 Removing ./ from despray

miguelvazq opened the pull request:
Fixing a commit https://github.com/hpcc-systems/HPCC-Platform/pull/8407/files that was overwriten with another commit.

Signed-off by: Miguel Vazquez miguel.vazquez@lexisnexis.com

miguelvazq commented:
@GordonSmith please review.

HPCCSmoketest commented:
Automated Smoketest 
Sha: 1fb6d0ce0b0e3d87e122f1dedc936ebbf42dd3e5
Build: success
Install hpccsystems-platform-community_6.0.0-rc1.el6.x86_64.rpm
HPCC Start: OK
HPCC Stop: OK
HPCC Uninstall: OK

GordonSmith commented:
Looks fine

AttilaVamos commented on esp/src/eclwatch/TargetSelectClass.js:
```diff
@@ -239,7 +240,7 @@ define([
             var context = this;
             this.getDropZoneFolder = function () {
-                var baseFolder = this._dropZoneTarget.machine.Directory + (this.endsWith(this._dropZoneTarget.machine.Directory, "/") ? "" : "/");
+                var baseFolder = this._dropZoneTarget.machine.Directory + (this.endsWith(this._dropZoneTarget.machine.Directory, "/") ? "" : "");
```
Maybe I am wrong, but does the second argument of this addition have any sense?

GordonSmith replied:
Yup, I agree

miguelvazq replied:
I'll take a look

Commit:
HPCC-15301 Removing ./ from despray

Fixing a commit https://github.com/hpcc-systems/HPCC-Platform/pull/8407/files that was overwriten.

Signed-off by: Miguel Vazquez <miguel.vazquez@lexisnexis.com>

miguelvazq commented:
@GordonSmith I removed the endswith call as it is not needed any longer. I also added a `required:false` to the LFDetailsWidget as there is a despray that can occur there as well.

HPCCSmoketest commented:
Automated Smoketest 
Sha: 7584c37f63bee954ee38137aad1ba2f4eef9e0c5
Build: success
Install hpccsystems-platform-community_6.0.0-rc1.el6.x86_64.rpm
HPCC Start: OK
HPCC Stop: OK
HPCC Uninstall: OK

GordonSmith commented:
Looks fine.
